release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -379,6 +402,45 @@ The Node Feature Discovery (NFD) Operator detects hardware features available on
 
 You can use the new `APIRequestCount` API to track what is using the deprecated APIs. This allows you to plan whether any actions are required in order to upgrade to the next release.
 
+[id="ocp-4-8-monitoring-stack-dependency-version-updates"]
+==== Version updates to monitoring stack components and dependencies
+
+{product-title} 4.8 includes version updates to the following monitoring stack components and dependencies:
+
+* The Prometheus Operator is now on version 0.48.1.
+* Prometheus is now on version 2.26.1.
+* The `node-exporter` agent is now on version 1.1.2.
```

Comment:
`node-exporter`
should be
node-exporter

Reply:
Going to keep the backticks on `node-exporter` because this is the style we use in the Monitoring docs.